Dockerfile

```diff
@@ -11,11 +11,21 @@ RUN cd \
   && curl -sS https://getcomposer.org/installer | php \
   && ln -s /root/composer.phar /usr/local/bin/composer
 
+
+FROM common as dev
```

Comment:
Jak tohle funguje?

Reply:
to je `target`, který pak můžeš použít v docker-compose.yml -> https://github.com/keboola/kbc-manage-api-php-client/blob/993b49b32ad4e5372487f2cf002856610c1a4bfd/docker-compose.yml#L40

Je to dobrý na to, když máš jeden Dockerfile a chceš mít možnost ho mít ve "více verzích". Tady je verze `composer install --dev` a `composer install` (bez dev závislostí).

Pro `docker build` se uplatní vždycky ta poslední verze. V tomhle případě `prod`

Reply:
Hustý, neznal jsem, díky!

Reply:
(resp. znal jsem to jen pro multistage build, ale ne takhle)

Reply:
Když se v tom už hraběš, tak by bylo fajn přidat rovnou i dvoukrokový composer install z php-component a dockerignore na vendor...

Reply:
v další PR ? :-)

Reply:
Když si lokálně zbuildnu testy, tak je to dost nekonečné, protože to bez cache 2x instaluje composer :(